app/src/main/java/fr/free/nrw/commons/campaigns/Campaign.java

```diff
@@ -0,0 +1,52 @@
+package fr.free.nrw.commons.campaigns;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Campaign {
```

Comment:
Please add Javadocs for all the new classes and methods. :) I guess there isn't really a need to have them for every single getter and setter method in this particular class (since they are fairly self-explanatory), but there should still at least be a Javadoc for the class itself. And the methods in the other classes would need them.

Reply:
yes I waa gonna, have added a lot of classes :P, will do it

Reply:
Done